src/fidesops/service/privacy_request/request_runner_service.py

```diff
@@ -192,8 +192,11 @@ def run(
 
             except BaseException as exc:  # pylint: disable=broad-except
                 privacy_request.status = PrivacyRequestStatus.error
+                privacy_request.save(db=session)
                 # If dev mode, log traceback
                 _log_exception(exc, config.dev_mode)
+                session.close()
+                return
```

Comment:
This will stop the post processing webhooks running when the request errors out, which is an interesting nuance that I don't think we've thought about at a product level. For example, in the event that webhooks are used to teardown infra (a requested case previously in Atlas), this would have the implication of that teardown work not being done. This may also mean we need to reconsider the pre-execution hooks in the event of a resume too. @adriaaaa do we have a preference here?

Reply:
If we decide to leave this, then we no longer need the check on line `:212`

Reply:
From offline discussion, we'll go with the current design - we'll assume webhooks should just run once. In the future, we can build out fancier webhooks that encompass a single collection or only run on failure, or only run on pause, for example.

Removed the check on L212.